.gitignore

```diff
@@ -55,6 +55,8 @@ coverage.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+.pytype
+.pyre
 
```

Comment:
I guess `.mypy_cache` should also be moved here from the end of the file.
Additionally it could be a good opportunity to remove some of the unnecessary settings here, although it's not at all important.